net/mstpd/files/etc/init.d/mstpd.init

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh /etc/rc.common
```

Comment:
I'd run through shellcheck. On first glance it seems there are some bashisms here.

Reply:
ran it, and added it in a separate commit, that I can squash later;
not all shellcheck complaints have been addressed though;
the ones that I don't want to get rid of is `In POSIX sh, 'local' is undefined.`

reason is: `ash` supports `local` keyword;
i've found [some time ago] that if you don't use `local`, you can get a crash, because shell vars don't get free'd from some internal table of vars of the shell interpreter [this probably also depends on how you write your shell code];
since OpenWrt uses UCI (which is a lot of shell magic) and you define 200 bridges or pseudo-bridges, you can get into such situations where you could [probably] still get a crash;

i've rewritten some of the parts that were easy picks;
and i just noticed that I can use `list_contains` instead of `mstpd_bridge_is_list`

Reply:
Right. Changing it to /bin/dash brings better shellcheck results.